web/app/fr/gospeak/web/auth/services/AuthSrv.scala

```diff
@@ -110,18 +112,65 @@ class AuthSrv(userRepo: AuthUserRepo,
     } yield authUser
   }
 
+  def createOrEdit(socialProfile: CommonSocialProfile): IO[AuthUser] = {
```

Comment:
Why not take `SocialProfile` as parameter here? 
So you can have dedicated methods for data transformation, ex: `socialProfile.merge(user): User` to use in edit function